snapcraft/__init__.py

```diff
@@ -138,15 +138,15 @@
 
   - build-packages: [deb, deb, deb...]
 
-    A list of Ubuntu packages to install on the build host before building
+    A list of packages to install on the build host before building
```

Comment:
Right, if we want to be generic, this messaging has to be generic. Which is why I also updated the errors mentioning Ubuntu (given that we already work on debian too :wink:)